spec/commands/v2/unpublish_spec.rb

```diff
@@ -374,6 +374,12 @@
 
       include_examples "creates an action"
 
+      it "maintains the state of unpublished" do
+        described_class.call(payload)
+        unpublished_content_item.reload
+        expect(unpublished_content_item.state).to eq("unpublished")
```

Comment:
could just do `expect(unpublished_content_item.reload.state).to eq("unpublished")` to drop the line above